pkg/server/server.go

```diff
@@ -669,6 +671,67 @@ func (s *Server) handleKnativeChannelCRDs(w http.ResponseWriter, r *http.Request
 	s.KnativeChannelCRDLister.HandleResources(w, r)
 }
 
+// buildCSPDirectives takes the content security policy configuration from the server and constructs
+// a complete set of directives for the Content-Security-Policy-Report-Only header.
+// The constructed directives will include the default sources and the supplied configuration.
+// buildCSPDirectives takes the content security policy configuration from the server and constructs
+// a complete set of directives for the Content-Security-Policy-Report-Only header.
+// The constructed directives will include the default sources and the supplied configuration.
```

Comment:
nit, repeated description
```suggestion
// buildCSPDirectives takes the content security policy configuration from the server and constructs
// a complete set of directives for the Content-Security-Policy-Report-Only header.
// The constructed directives will include the default sources and the supplied configuration.
```